.github/workflows/mobilecoin-workflow-dev-deploy.yaml

```diff
@@ -247,6 +247,54 @@ jobs:
         rancher_url: ${{ secrets.DEV_RANCHER_URL }}
         rancher_token: ${{ secrets.DEV_RANCHER_TOKEN }}
 
+  fog-view-deploy:
+    needs:
+    - consensus-deploy
+    runs-on: [self-hosted, Linux, small]
+    steps:
+    - name: Generate fog-view values file
+      run: |
+        mkdir -p "${VALUES_BASE_PATH}"
+        cat <<EOF > "${VALUES_BASE_PATH}/fog-view-values.yaml"
+        image:
+          org: ${{ inputs.docker_image_org }}
+
+        mobilecoin:
+          network: ${{ inputs.namespace }}
+          partner: dev
+
+        fogView:
+          router:
+            hosts:
+            - partner: a
+              responderID: fog.${{ inputs.namespace }}.development.mobilecoin.com
+            - partner: b
+              responderID: fog-b.${{ inputs.namespace }}.development.mobilecoin.com
+          ingress:
```

Comment:
Pattern for adding multiple router instances with different responder ids.  Creates aligned service, ingress, and certificates.